src/router/index.js

```diff
@@ -1,12 +1,17 @@
 import { createRouter, createWebHistory } from 'vue-router'
-import TabbedView from '../views/TabbedView.vue'
+import TabbedView from '@/views/TabbedView.vue'
 
 const routes = [
   {
     path: '/',
     name: 'TabbedView',
     component: TabbedView
   },
+  {
+    path: '/search-pokemons',
+    name: 'SearchView',
+    component: () => import('@/views/SearchView')
```

Comment:
It is better to use a consistent syntax for importing throughout the route config.

Reply:
I am not sure I understand this?

Reply:
import SearchView from '@/views/SearchView.vue'
    
And then: 
    
    component: SearchView
    
So the syntax is consistent with the TabbedView

Reply:
I think you are wrong here, beacuse this is an async import, meaning it will only load the component when the route is visted( lazy loading mechanism) thus keeping the main bundle smaller 
https://router.vuejs.org/guide/advanced/lazy-loading.html